tests/output_guardrails/test_cli.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+import yaml
+from click.testing import CliRunner
+
+from govuk_chat_evaluation.output_guardrails.cli import main, Config
+from govuk_chat_evaluation.output_guardrails.evaluate import EvaluationResult
+
+
+class TestConfig:
+    def test_config_requires_provider_for_generate(self, mock_input_data):
+        with pytest.raises(ValueError, match="provider is required to generate data"):
+            Config(
+                what="Test",
+                guardrail_type="answer_guardrails",
+                generate=True,
+                provider=None,
+                input_path=mock_input_data,
+            )
+
+        Config(
+            what="Test",
+            guardrail_type="answer_guardrails",
+            generate=False,
+            provider=None,
+            input_path=mock_input_data,
+        )
+
+        Config(
+            what="Test",
+            guardrail_type="answer_guardrails",
+            generate=True,
+            provider="openai",
+            input_path=mock_input_data,
+        )
+
+
+@pytest.fixture(autouse=True)
+def mock_config_file(tmp_path, mock_input_data):
+    """Write a config file as an input for testing"""
+    data = {
+        "what": "Testing Ooutput Guardrail evaluations",
+        "guardrail_type": "answer_guardrails",
+        "generate": False,
```

Comment:
```suggestion
        "generate": True,
```

I think we should probably set this to true for the tests, it seems to create a more honest scenario for no-generate (another thing to fix in jailbreak guardrails)